lib/sms_autofill.dart

```diff
@@ -332,8 +333,8 @@ class _PhoneFieldHintState extends State<_PhoneFieldHint> {
   }
 
   Future<void> _askPhoneHint() async {
-    String hint = await _autoFill.hint;
-    _controller.value = TextEditingValue(text: hint ?? '');
+    String? hint = await _autoFill.hint;
+    _controller.value = TextEditingValue(text: hint);
```

Comment:
`TextEditingValue` allow you to put `text` as nullable ??

Reply:
Its starts giving a warning if I add a hint??" ".